src/spec/base.ts

```diff
@@ -72,29 +72,31 @@ export interface LayoutSizeMixins {
    *
    * - For a plot with a continuous x-field, width should be a number.
    * - For a plot with either a discrete x-field or no x-field, width can be either a number indicating a fixed width or an object in the form of `{step: number}` defining the width per discrete step. (No x-field is equivalent to having one discrete step.)
+   * - To enable responsive sizing on width, it should be set to `container`.
```

Comment:
Please use quotes inside backticks for string values

```
`"container"`
```

(We can omit quotes for property name just like in JS)

Reply:
Good catch.